tests/integration-tests/tests/schedulers/test_slurm/test_slurm_config_update/pcluster.config.update.yaml

```diff
@@ -8,7 +8,7 @@ HeadNode:
     KeyName: {{ key_name }}
   Iam:
     AdditionalIamPolicies:
-      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+      - Policy: arn:{{partition}}:iam::aws:policy/AmazonSSMManagedInstanceCore
```

Comment:
We should parameterize the partition in this way in many other yaml file I suppose.
For example: I see a policy ARN `arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess` also in `/Users/mgiacomo/workplace/aws-parallelcluster-dev/aws-parallelcluster/cli/tests/pcluster3_config_converter/test_pcluster3_config_converter/test_pcluster3_config_converter/slurm_full.yaml`.

Let's fix the ARN issue everywhere in integ tests

Reply:
The ones you mentioned are from unit tests. We will keep them unchanged because this PR focuses on integration tests

Reply:
Ok